src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -168,6 +182,16 @@ public Action<IApplicationBuilder> Configure(Action<IApplicationBuilder> next)
 
                     app.UseRequestContextAfterAuthentication<IDicomRequestContext>();
 
+                    app.UseSwagger();
+
+                    app.UseSwaggerUI(options =>
+                    {
+                        foreach (var description in provider.ApiVersionDescriptions)
+                        {
+                            options.SwaggerEndpoint($"/swagger/{description.GroupName}/swagger.json", description.GroupName.ToUpperInvariant());
```

Comment:
I think after their rename (rebranding?), "Swagger" refers to the tooling whereas "OpenAPI" refers to the specification.

Should we move the version-specific OpenAPI documents to existing versioned routes as opposed to their own routes like `"/{description.GroupName}/api.json"`? I suppose a better question too is: is there a standard convention for exposing these documents and swagger ui?

Reply:
We looked into it a bit, couldnt find any standard and seems like most apis just have their own preference. I left it how it was for now but if you have any preference I can update it.

Reply:
Perhaps:
- `/api/swagger` - the Swagger UI
- `/api/{version}.yaml` - the OpenAPI specification for the given `version`

It seems this UI is powered by a static file, but I wish it was something whose format, either JSON or YAML, could be converted at runtime. Oh well!